apps/infrastructure/src/ecs.tf

```diff
@@ -56,19 +57,18 @@ resource "aws_ecs_task_definition" "cms_task_def" {
 module "cms_ecs_service" {
   source = "git::https://github.com/terraform-aws-modules/terraform-aws-ecs.git//modules/service?ref=8b97783def49997d18a6fcb00dc21ce1edc0f538" # v5.9.0
 
-  name                           = "cms-ecs"
-  cluster_arn                    = module.cms_ecs_cluster.arn
-  desired_count                  = 1
-  create_task_definition         = false
-  create_iam_role                = false
-  create_task_exec_iam_role      = false
-  create_security_group          = false
-  launch_type                    = "FARGATE"
-  force_new_deployment           = true
-  task_definition_arn            = aws_ecs_task_definition.cms_task_def.arn
-  tasks_iam_role_arn             = module.iam_role_task_role.iam_role_arn
-  task_exec_iam_role_arn         = module.iam_role_ecs_task_execution.iam_role_arn
-  ignore_task_definition_changes = true # CMS Deployment is managed by the "Deploy CMS" GitHub Action
+  name                      = "cms-ecs"
+  cluster_arn               = module.cms_ecs_cluster.arn
+  desired_count             = var.environment == "dev" ? 1 : 0
+  create_task_definition    = false
+  create_iam_role           = false
+  create_task_exec_iam_role = false
+  create_security_group     = false
+  launch_type               = "FARGATE"
+  force_new_deployment      = true
+  task_definition_arn       = aws_ecs_task_definition.cms_task_def.arn
+  tasks_iam_role_arn        = module.iam_role_task_role.iam_role_arn
+  task_exec_iam_role_arn    = module.iam_role_ecs_task_execution.iam_role_arn
```

Comment:
pay attention here, because you are introducing a regression. [take a look at the very same file on `main` branch](https://github.com/pagopa/developer-portal/blob/f54c1418358056243755137e81df66254c502c00/apps/infrastructure/src/ecs.tf#L59-L71)